azure-cli.js

```diff
@@ -28,6 +56,19 @@ async function execute({ command, credentials }) {
   }
 }
 
+async function createVolumeConfigsWithPathMatch(command) {
+  const pathMatches = kaholoPluginLibrary.helpers.extractPathsFromCommand(command);
+
+  await validatePaths(pathMatches.map(({ path }) => path));
+
+  return pathMatches.map(
```

Comment:
A function/method should do one thing, this one seems to do two:
- extract paths
- create docker volume configs

Problem with naming such function and then extra processing (mapping) its result in the `execute` emphasizes that there's something wrong.